console/src/view/help.rs

```diff
@@ -53,15 +59,28 @@ where
             )
             .split(popup_layout[1])[1];
 
-        // TODO: This doesn't have to be a clone
-        let content = self
-            .help_text
-            .take()
-            .expect("help_text should be initialized");
+        let mut height = 1;
+        let width = content.width() as u16;
+        if popup_area.width < width {
+            height = width / popup_area.width;
+
+            if width % popup_area.width > 0 {
+                height += 1
+            }
+        }
+
+        let content_layout = layout::Layout::default()
+            .direction(layout::Direction::Vertical)
+            .margin(0);
+
+        let content_area = content_layout
+            .constraints([layout::Constraint::Length(height)])
```

Comment:
I wanted to create an area within the popup to render the text into, which would be responsive, but doing it like this doesn't seem to be working.

Reply:
what isn't working, exactly? is the problem that the text isn't wrapping?